.env.example

```
@@ -1,30 +1,24 @@
-# Copy to .env — the server loads it on boot.
-
 PORT=3400
-JWT_SECRET=
-DATABASE_URL=
+JWT_SECRET=verysecrettoken
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_SECRET key should go before the PORT key

(UnorderedKey)